config/subsample_genbank.yaml

```diff
@@ -0,0 +1,19 @@
+# All the config parameters needed to run ncov-ingest for GenBank sequences
+# without all the bells and whistles used by internal Nextstrain runs that
+# involve AWS S3, Slack notifications, and GitHub Action triggers.
+database_name: "genbank"
+
+s3_src: "s3://nextstrain-data/files/ncov/open"
+s3_dst: "s3://nextstrain-data/files/ncov/open/branch/subsample"
+
+keep_all_files: True
+fetch_from_database: false
+trigger_rebuild: false
+trigger_counts: false
+
+# How many lines to extract from s3 fetch
+subsample:
+  main_ndjson: 1000
+  biosample: 100000
+  nextclade: 10000
```

Comment:
I'm slightly confused by the nextclade subsampling. Are you leaving it up to chance of how many of the `main_ndjson` records will be in the subsampled nextclade files?

Reply:
Yes exactly - numbers are big enough so that by chance we definitely have _some_ previously aligned sequences there. All paths are tested but the output data should not be trusted of course.

This also applies to the biosample: random biosample records are chosen, not matched to main_ndjson.

I'll describe it in dev docs.